s3_object_life_cycle.sh

```diff
@@ -0,0 +1,15 @@
+# set object lifecycle for openstates-realtime-bills s3 bucket: deletion after 2 days
+aws s3api put-lifecycle-configuration --bucket openstates-realtime-bills --lifecycle-configuration '{
```

Comment:
If we're going to put this in a script, I'd like to see a proper `#!` at the beginning and some effort around ensuring that packages exist/etc.
Otherwise, we add this command in the README with documentation around it explaining _why_ we add a lifecycle policy.